### ISSUE TITLE
With 'sexy' theme kernel 'make menuconfig' is garbled.

### ISSUE THREAD

**Issue by AlexanderAmelkin** [Bash-it/bash-it#822]
Somehow when I use 'sexy' theme with bash_it, 'menuconfig' in Linux kernel gets broken:

![garbled_menuconfig](https://cloud.githubusercontent.com/assets/4518124/19972367/da6a332e-a1f2-11e6-98a6-52d8ce56f4c6.png)

**Comment by nwinkler** [comment]
What does it look like without Bash-it or with a different theme?

**Comment by AlexanderAmelkin** [comment]
In a few words: it looks normal. ;) That is, without black areas.
This garbling also happens with 'axin' theme, but doesn't happen with 'mairan', althout both 'axin' and 'sexy' seem to be based on 'mairan' (at least they have most common lines with that theme).

![normal-menuconfig](https://cloud.githubusercontent.com/assets/4518124/19973942/d3655a5e-a1f7-11e6-9d81-cae03195f2f3.png)

**Comment by nwinkler** [comment]
Well, both of the problematic ones have these lines (https://github.com/Bash-it/bash-it/blob/master/themes/sexy/sexy.theme.bash#L5):

```
if [[ $COLORTERM = gnome-* && $TERM = xterm ]]  && infocmp gnome-256color >/dev/null 2>&1; then export TERM=gnome-256color
elif [[ $TERM != dumb ]] && infocmp xterm-256color >/dev/null 2>&1; then export TERM=xterm-256color
fi
```

Can you try commenting out this block from one of the themes and then check whether it's fixing the issue?

**Comment by AlexanderAmelkin** [comment]
I just realized I was running inside a screen session, so I thought that it could have been inheriting some environment information from the parent session, so my changes to theme selection could have been... ehm... spoiled.

I now tried a completely new session and I don't see those black regions with neither sexy, nor axin themes. However as soon as I start 'screen', I start seeing the black regions again regardless of commenting out the lines you mentioned.

I tried changing the TERM to screen.putty-256color manually in a non-screen session. Black regions do not appear.

Attached are my environments from a bare session (good.env) and from a screen session (bad.env) that I started by simply running 'screen' from within 'good.env'.

[environments.tar.gz](https://github.com/Bash-it/bash-it/files/575077/environments.tar.gz)
